projects/playground/src/app/app.module.ts

```diff
@@ -27,9 +41,16 @@ const AUTH0_CONFIG = {
     BrowserModule,
     AppRoutingModule,
     ReactiveFormsModule,
-    AuthModule.forRoot(AUTH0_CONFIG),
+    AuthModule.forRoot(/* authConfig */), // Uncomment this to specify config directly
+  ],
+  providers: [
+    {
+      provide: APP_INITIALIZER,
```

Comment:
As far as I understand, if forRoot() is called with an authConfig then the APP_INITIALIZER provider is optional(?).

This detail is described in the PR description.

Perhaps the comment on :44 could be supported by a comment detailing provider implementation.

Reply:
I think adding a comment to indicate you should use one way or the other is a great idea. I would stay away from introducing another sample only for the sake of demonstrating this.